ui/selectors/selectors.js

```diff
@@ -603,6 +604,18 @@ export function getShowTestNetworks(state) {
   return Boolean(showTestNetworks);
 }
 
+export function getTestNetworkBackgroundColor(state) {
+  const currentNetwork = state.metamask.providerConfig.ticker;
+  switch (true) {
+    case currentNetwork?.includes(GOERLI_DISPLAY_NAME):
+      return BackgroundColor.goerli;
+    case currentNetwork?.includes(SEPOLIA_DISPLAY_NAME):
+      return BackgroundColor.sepolia;
+    default:
+      return undefined;
```

Comment:
Should we have a default backgroundColor like white or something, so that we won't have component render undefined color (in case the component doesn't deal with undefined)

Reply:
Umm, for the rest of the networks if we have a logo, we show it and if they don't have it, we show the first letter of the name with the AvatarNetwork default background color.